src/detection/DepthImagePolarHistDetector.cc

```diff
@@ -101,7 +101,17 @@ const std::vector<Obstacle> &DepthImagePolarHistDetector::detect(
         double phi = max_phi - (i * fixed_step) - (fixed_step / 2);
 
         Obstacle obs;
+
+        obs.bounding_box.tlc.x = histogram[i];
+        obs.bounding_box.tlc.y = (M_PI - depth_data->vfov) / 2 ;
```

Comment:
> This seems to be swapped. If if the center is at M_PI / 2, left of the center should be bigger than that, not smaller.
But tlc.y defines the top, not the left. Our Z points up, so Polar Angle = 0 degrees points straight up. I wanna point to the top of the vertical field of view, so that would be M_PI / 2 - vfov / 2, so I think that the calculation is right, isn't? Does this makes sense to you know?

> Also, if you add vfov/2 to both sizes, isn't the obstacle covering the whole sensor view? seems a bit too big.
Because the way Polar histogram detects obstacle (IOW we don't know where the obstacle on a given slice is on the vertical of that slice, but only that the slice contains an obstacle), the bounding of the obstacle end up being the bounds of the slice itself, so indeed it covers the entire vertical field of view.